.circleci/config.yml

```diff
@@ -78,6 +71,18 @@ commands:
           name: Run JavaScript tests
           command: npm run build:css && npm run test:ci
 
+  run_python_tests:
+    steps:
+      - run:
+          name: Run Python tests
+          command: make pytest
+
+  run_e2e_tests:
+    steps:
+      - run:
+          name: Run all end to end tests
+          command: make e2e-tests
+
```

Comment:
These can go straight into the relevant `job`. No need to define it as a separate command.

Reply:
Similarly for several of the other commands actually. Basic rule: if a `command` is just one `run` step long and only used in one `job`, there's no point having a `command` for it. Just put it directly into the `job` instead.